app/views/main/login.html.erb

```diff
@@ -1,5 +1,20 @@
 <% content_for :head do %>
   <%= javascript_include_tag 'cookies_eu', nonce: true %>
+  <%= javascript_tag nonce: true do %>
+    document.addEventListener('DOMContentLoaded', () => {
+      document.getElementById('local-login-selector').addEventListener('click', () => {
+        const formElem = document.getElementById('local-login-form');
+        const orElem = document.getElementById('login-or-container');
```

Comment:
Maybe this is picky but I'd like to see the "OR" div always visible when both forms of authentication is supported, so that the "expanded MarkUs login" view looks like 

![image](https://user-images.githubusercontent.com/3333115/149249449-7d8e8c57-cee7-4c83-8058-7afebdb3a9e5.png)

I guess this makes the HTML structure below slightly more complex since you need an extra conditional for the OR, but I think the UI looks a bit better that way. :)